pulp_rpm/app/kickstart/treeinfo.py

```diff
@@ -246,9 +115,26 @@ def parsed_sections(self):
         parser = SortedConfigParser()
         self.serialize(parser)
 
-        if "general" in self.original_parser._sections:
+        general = self.original_parser._sections.get("general")
+        build_timestamp = ""
+
+        if general:
             if "general" not in parser._sections:
-                parser._sections["general"] = self.original_parser._sections["general"]
+                parser._sections["general"] = general
+
+            if "." in str(general.get("timestamp")):
+                build_timestamp = float(general["timestamp"])
+
+        tree = self.original_parser._sections.get("tree", {})
+
+        if "." in str(tree.get("build_timestamp")):
+            build_timestamp = float(tree.get["build_timestamp"])
+
+        if build_timestamp and parser._sections.get("general", {}).get("timestamp"):
+            parser._sections["general"]["timestamp"] = build_timestamp
+
+        if build_timestamp and parser._sections.get("tree", {}).get("build_timestamp"):
+            parser._sections["tree"]["build_timestamp"] = build_timestamp
```

Comment:
Could general timestamp and tree build timestamp ever be different?

Reply:
If so, it seems like this code would set both to the same value?

Reply:
from what I understood from https://productmd.readthedocs.io/en/latest/treeinfo-1.0.html
[general] is built gathering information from other sections, so I think it could not be different